tests/devices/qubit/test_simulate.py

```diff
@@ -0,0 +1,583 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for create_initial_state in devices/qubit."""
+
+import pytest
+
+import numpy as np
+
+import pennylane as qml
+from pennylane.devices.qubit import simulate
+
+ml_frameworks_list = [
+    "numpy",
+    pytest.param("autograd", marks=pytest.mark.autograd),
+    pytest.param("jax", marks=pytest.mark.jax),
+    pytest.param("torch", marks=pytest.mark.torch),
+    pytest.param("tensorflow", marks=pytest.mark.tf),
+]
```

Comment:
What is this? 😮

Reply:
A way to parametrize over interfaces but have them still marked.

Reply:
Actually, looks like I didn't actually end up using this after all because they all have different ways to take derivatives.

```suggestion
```